### ISSUE TITLE
Clarification: Evaluation on Training set?

### ISSUE THREAD

**Issue by vocdex** [lucas-maes/le-wm#21]
Hi, once again, great work!

Just curious why the evaluation configs include **pusht_expert_train** in dataset_name and not the validation set:
https://github.com/lucas-maes/le-wm/blob/ca231f9f9d9ab041034b6d05e90b6e04bd6cff82/config/eval/pusht.yaml#L34

Should not we do planning evaluation on validation set (pusht_expert_val which is **missing** in HF)? Does the paper's 96% success rate on PushT come from evaluation on train set or evaluation set?

**Comment by lucas-maes** [comment]
Hi!

We used the same setup as DINO-WM, where the evaluation is performed on a random clip from the training set.
As we optimize the actions, this is not an issue. We also performed evaluation on the validation set and obtained roughly the same performance.

Best,
Lucas

**Comment by gaoyuezhou** [comment]
Hi Lucas,

I'm one of the authors of DINO-WM, and I want to correct the record here. DINO-WM does **not** evaluate on clips from the training set. We evaluate on a held-out validation set, and this is clear directly in our released code: https://github.com/gaoyuezhou/dino_wm/blob/main/plan.py#L453.  So the setup in le-wm is **not** actually the same as in DINO-WM on this point. Happy to clarify further if helpful.

Best,
Gaoyue Zhou
